Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -1242,12 +1241,8 @@ def get_fetch_times_and_offset(incident_type):
 
 
 def fetch_incidents():
-    """
-        Fetches incident using the detections API
-        :return: Fetched detections in incident format
-    """
     incidents = []  # type:List
-
+    demisto_set_last_run = {}
```

Comment:
```suggestion
    current_fetch_info = {}
```

This is just a suggestion.. just that demisto_set_last_run does not really indicate what this object really does.

Reply:
done